tests/benchmark/tofu.go

```diff
@@ -88,6 +95,7 @@ func runTofuBenchmarks(testCases []testCase) map[string]*benchmarkResult {
 			results[tc.name].planSuccess = true
 		}
 
+		defer runTofuDestroy(dir)
```

Comment:
What's the practical difference between running destroy here vs where it used to be?